app/api/models/Events.py

```diff
@@ -92,11 +92,17 @@ def rsvpEvent(self, user_id, event_id, clientEmail):
         """
```

Comment:
python files/modules names are usually all in small caps, so instead of Events.py, the file should be named events.py
Only class names should begin with a capital letter.